css-animations-2/Overview.bs

```diff
@@ -557,13 +557,27 @@ the following effects:
 ::  The animation is not associated with a [=timeline=].
 
 :   <dfn><<timeline-name>></dfn>
-::  Find the last timeline at-rule in document order with its name matching
+::  If a named [=scroll progress timeline=] or [=view progress timeline=]
+    is in scope on this element,
+    use the referenced timeline
+    as defined in [[scroll-animations-1#timeline-scope]].
+
+    Otherwise, find the last timeline at-rule in document order with its name matching
```

Comment:
I believe scroll timelines were the only at-rule timelines so with those gone we can probably remove the at-rule matching.

Reply:
Done.